app/server/utils/metrics/metric.py

```diff
@@ -66,6 +66,16 @@ def execute_query(self, user_filters: dict = None, date_filters_dict=None, enabl
 
         if self.is_timeseries:
             result = {}
+            if self.value_type not in VALUE_TYPES:
+                raise Exception(f'{self.value_type} not a valid metric type!')
+            result['type'] = {
+                'type': self.value_type,
+                'display_decimals': 2 if self.value_type == COUNT_AVERAGE else 0
+            }
+            if self.token:
+                result['type']['currency_name'] = self.token.name
+                result['type']['currency_symbol'] = self.token.symbol
+                result['type']['display_decimals'] = 2 # change this to self.token.display_decimals once #251 is merged
```

Comment:
👍

Reply:
For the tests, could we have a token with `display_tokens=0`  and `display_tokens=2`. We'll have both these usecases live with USD and VT (vatu)

Reply:
Added a test! ✅